django/db/backends/mysql/features.py

```diff
@@ -178,7 +178,9 @@ def can_introspect_check_constraints(self):
 
     @cached_property
     def has_select_for_update_skip_locked(self):
-        return not self.connection.mysql_is_mariadb and self.connection.mysql_version >= (8, 0, 1)
+        if self.connection.mysql_is_mariadb:
+            return self.connection.mysql_version >= (10, 6)
+        return self.connection.mysql_version >= (8, 0, 1)
```

Comment:
Can we do it like:
```suggestion
        return (
            (self.connection.mysql_is_mariadb and self.connection.mysql_version >= (10, 6)) or 
            (self.connection.mysql_version >= (8, 0, 1)
        )
```

Reply:
No, because it would return `True` for MariaDB < 10.6.